src/renderers/webgpu/nodes/WGSLNodeBuilder.js

```diff
@@ -1899,6 +1899,21 @@ ${ flowData.code }
 
 	}
 
+	/**
+	 * Returns the native snippet for a ternary operation.
+	 *
+	 * @param {string} condSnippet - The condition determining which expression gets resolved.
+	 * @param {string} ifSnippet - The expression to resolve to if the condition is true.
+	 * @param {string} elseSnippet - The expression to resolve to if the condition is false.
+	 * @return {string} The resolved method name.
+	 */
+	getTernary( condSnippet, ifSnippet, elseSnippet ) {
+
+		return `select(${elseSnippet}, ${ifSnippet}, ${condSnippet})`;
```

Comment:
It's best separate with spaces `select( ${ elseSnippet }, ${ ifSnippet }, ${ condSnippet } )`;